src/monitoring_service/service.py

```diff
@@ -33,7 +33,7 @@
 def check_gas_reserve(web3: Web3, private_key: str) -> None:
     """ Check periodically for gas reserve in the account """
     gas_price = web3.eth.gasPrice
-    gas_limit = GAS_REQUIRED_FOR_MS_MONITOR
+    gas_limit = gas_measurements()["MonitoringService.monitor"]
```

Comment:
Should we make the `gas_measurements()` a global like the contracts manager? Otherwise we might forget to update this call when adding the version?

Reply:
If we don't specify the version it uses the latest code, which sounds just like what we want to do.

Reply:
Sure, we just need to remember to add the version in thswi place as well.